installation-bundle/src/Database/Installer.php

```diff
@@ -235,7 +235,11 @@ private function checkEngineAndCollation(array &$sql, Schema $toSchema): void
             if ($deleteIndexes) {
                 $platform = $this->connection->getDatabasePlatform();
 
-                foreach ($table->getIndexes() as $index) {
+                if (!$fromSchema->hasTable($tableName)) {
+                    continue;
+                }
+
+                foreach ($fromSchema->getTable($tableName)->getIndexes() as $index) {
                     if ('primary' !== $index->getName()) {
```

Comment:
This needs to be `strtolower($index->getName())` to fix https://github.com/contao/contao/issues/140#issuecomment-436284620.